gadget/layout.go

```diff
@@ -226,6 +256,15 @@ func LayoutVolume(gadgetRootDir string, volume *Volume, constraints LayoutConstr
 		}
 
 		structures[idx].LaidOutContent = content
+
+		// resolve filesystem content
+		if !constraints.SkipResolveContent {
```

Comment:
```suggestion
		if !constraints.SkipResolveContent && ps.HasFilessytem() {
```

Reply:
I'm happy to change it but AIUI I'm following the pattern of layOutStructureContent() which also is called unconditionally.